docs/versioned_docs/version-zenith/user/quickstart/562821-hello.mdx

```diff
@@ -0,0 +1,59 @@
+---
+slug: /user/quickstart/562821/hello
+hide_table_of_contents: true
+title: "Create your first Dagger module"
+---
+import PartialExperimentalDocs from '../../partials/_experimental.mdx';
+
+# Quickstart
+
+<PartialExperimentalDocs />
+
+## Create your first Dagger module
+
+You're now ready to dive into Dagger and write your first module!
+
+This first example is fairly simple: it creates a Dagger module that initializes a Node.js container, executes a command in that container, and prints the output.
+
+Begin by running `dagger mod init` in your application directory to bootstrap a new module using the Dagger Go SDK:
+
+```shell
+cd hello-dagger
+dagger mod init --name=mymodule --sdk=go
+```
+
+This will generate a `dagger.json` module file, an initial `main.go` source file, as well as a generated `dagger.gen.go` and `internal` folder for the generated module code.
+
+Update the generated `main.go` file with the following code:
+
+```go file=snippets/hello/main.go
+```
+
+This module defines a function named `Hello()` which returns a string. This function performs the following operations:
+
+- It uses the `dag` client's `Container().From()` method to initialize a new container from a base image. In this example, the base image is `node:18-slim`. This method returns a `Container` representing an OCI-compatible container image.
+- It uses the `Container.WithExec()` method to define the command to be executed in the container - in this case, the command `node -v`, which returns the Node version string. The `WithExec()` method returns a revised Container with the results of command execution.
+- It retrieves the output stream of the command with the `Container.Stdout()` method and assigns it to a variable. Since the `Container.Stdout()` method uses the `Context` type, the module imports the `context` package.
+- It returns a message interpolating the variable (the Node version number) into a longer string.
+
+:::info
+`dag` is the Dagger client, which is pre-initialized. It contains all the core types (like `Container`, `Directory`, etc.), as well as bindings to any dependencies your module has declared. Exhaustive documentation is available for core types and methods in the [Go SDK reference](https://pkg.go.dev/dagger.io/dagger).
+:::
+
+Call the new function using the Dagger CLI:
+
+```shell
+dagger call hello
+```
+
+:::info
+When using `dagger call`, all names (functions, arguments, fields, etc) are converted into a shell-friendly "kebab-case" style.
```

Comment:
nit: If we change the name of the function from `Hello` to `HelloNode` or similar then we can show this more concretely since the command would be `dagger call hello-node`

Reply:
Good idea, and I would make this language-agnostic, i.e. `HelloFromDagger`

Reply:
Done